### DISCUSSION TITLE
Document for Next.js v7.0.3

### DISCUSSION THREAD

**Discussion by ryota-murakami** [vercel/next.js#10683] [Help] [answered]
I'm really new to Next.js, and my working project using Next.js v7.0.3.
I couldn't find specific versions document, but I figure out the [old document](https://nextjs.org/docs/old) istead.

Is the old document available for Next.js v7.0.3?

Thank you 😀

**Comment by timneutkens** [comment, marked as answer]
You can use GitHub to find the code/docs from that version: https://github.com/zeit/next.js/tree/7.0.3

**Reply by ryota-murakami** [reply]
I got it, thank you so much!
